tests/unit/h/util/db_test.py

```diff
@@ -81,7 +81,7 @@ def test_skips_wrapped_function_when_nested_transaction(
 
         @on_transaction_end(db_session)
         def transaction_ended():
-            spy()
+            spy()  # pragma: nocover
```

Comment:
The test isn't actually meant to hit this line. If it does, the `assert` below will fail. Not sure how this ever worked without a `# pragma: nocover` since we were enforcing 100% coverage :man_shrugging: